templates/navigation.hbs

```diff
@@ -1,11 +1,11 @@
     <div class="nav-container">
       <div class="container">
-        <div class="pure-menu pure-menu-horizontal">
+        <div class="pure-menu pure-menu-horizontal" role="navigation" aria-label="Main navigation">
           <form action="/releases/search" method="GET" class="landing-search-form-nav">
             {{#unless varsb.show_search_form}}
-            <input class="search-input-nav" name="query" type="text" placeholder="Find crate"{{#if varss.search_query}} value="{{varss.search_query}}"{{/if}}>
+            <input class="search-input-nav" name="query" type="text" aria-label="Find crate by search query" placeholder="Find crate"{{#if varss.search_query}} value="{{varss.search_query}}"{{/if}}>
             {{/unless}}
-          <a href="/" class="pure-menu-heading pure-menu-link"><i class="fa fa-cubes fa-fw"></i> Docs.rs</a>
+          <a href="/" class="pure-menu-heading pure-menu-link" aria-label="Docs.rs"><i class="fa fa-cubes fa-fw"></i> Docs.rs</a>
```

Comment:
What about "Back to home page"?

Reply:
This link appears on most, if not all pages on docs.rs. I don't think it makes sense to label it as returning "back" to anywhere, since the user may have not previously been on the homepage. 

Is the current labeling acceptable?